sql/src/test/java/io/crate/executor/task/SetSessionTaskTest.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Licensed to Crate under one or more contributor license agreements.
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.  Crate licenses this file
+ * to you under the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.  You may
+ * obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied.  See the License for the specific language governing
+ * permissions and limitations under the License.
+ *
+ * However, if you have executed another commercial license agreement
+ * with Crate these terms will supersede the license and you may use the
+ * software solely pursuant to the terms of the relevant commercial
+ * agreement.
+ */
+
+package io.crate.executor.task;
+
+import com.google.common.collect.ImmutableList;
+import io.crate.action.sql.SessionContext;
+import io.crate.core.collections.Row;
+import io.crate.operation.projectors.RowReceiver;
+import io.crate.sql.parser.SqlParser;
+import io.crate.sql.tree.Expression;
+import io.crate.test.integration.CrateUnitTest;
+import org.junit.Test;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.UUID;
+
+import static org.mockito.Matchers.any;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+
+public class SetSessionTaskTest extends CrateUnitTest {
```

Comment:
ah ok, so if you seen no case at on how this could fail. then we don't need the try/catch and also not the test, since it would test a scenario which does actually not exist. just make sure this really is the case. did you test for example a parameter with null? like set...=? args=[null]

i just want to make sure that there is really no case here, should you remove the catch clause you should add a note why there cannot be any exception in this case.